application/app/services/dataverse/user_service.rb

```diff
@@ -0,0 +1,29 @@
+module Dataverse
+  class UserService
+    include LoggingCommon
+    include DateTimeCommon
+
+    AUTH_HEADER = 'X-Dataverse-key'
```

Comment:
We need to add an issue to refactor the DV clients to put these in a concern or base class

Reply:
ok

Reply:
I created the issue https://github.com/hmdc/DevOpsProjects/issues/1761